packages/core/src/types.ts

```diff
@@ -900,7 +1043,12 @@ export enum ActionTypes {
   ErrorCustom = 'xstate.error',
   Update = 'xstate.update',
   Pure = 'xstate.pure',
-  Choose = 'xstate.choose'
+  Choose = 'xstate.choose',
+  Input = 'xstate.input'
```

Comment:
as `EventTypes` has been created in this PR this is being redundant
```suggestion
  Choose = 'xstate.choose'
```